src/yellow/components/topbar/index.stories.js

```diff
@@ -0,0 +1,80 @@
+import Logo from '../../../../public/images/Logo.png';
+
+export default {
+  title: 'Components | Topbar',
+  parameters: {
+    componentSubtitle: 'Used in top of the pages in Website',
+  },
+}
+
+import './_index.scss';
+import '../heading/_index.scss';
+import '../quote/_index.scss';
+import '../button/_index.scss';
```

Comment:
Is there any reason to keep the imports in different parts of the file?

Reply:
There is no reason.

Reply:
I would suggest to change it to keep imports and exports together, but in other files already exists this structure. So it is better to keep this way.

Reply:
When we did that, we were thinking more about exporting Storybook settings and then importing the necessary files. Maybe this file was confusing, because I made an import before exporting the storybook settings.